### PR TITLE
Updates create_child_work_spec.rb for valkyrie.

### PR DESCRIPTION
**Pull request by bwatson78**
### Fixes

Fixes `create_child_work_spec.rb`.

### Summary

Updates create_child_work_spec.rb for valkyrie.

### Type of change (for release notes)

- `notes-valkyrie` Valkyrie Progress

### Changes proposed in this pull request:
* Swaps to valkyrie monographs for work objects.
* Switches to updated sipity entity factory proxy setting.
* Points elements to monograph paths.
* Removes any error messaging or object variables that are ActiveFedora specific.

@samvera/hyrax-code-reviewers

### CODE REVIEW

**Review by jeremyf — DISMISSED** [review]

File: spec/features/create_child_work_spec.rb [hunk 1]
```diff
@@ -22,61 +26,57 @@
...
+      persister.save(resource: parent)
     end
+
     it 'can be updated' do
-      visit "/concern/parent/#{parent.id}/generic_works/#{curation_concern.id}/edit"
+      visit "/concern/parent/#{parent.id}/monographs/#{curation_concern.id}/edit"
       click_on "Save"
 
-      expect(parent.reload.ordered_members.to_a.length).to eq 1
+      expect(parent.member_ids.count).to eq 1
```
Comment (dlpierce) [inline spec/features/create_child_work_spec.rb:55]:
```suggestion
      expect(Hyrax.query_service.find_by(id: parent.id).member_ids.count).to eq 1
```

File: spec/features/create_child_work_spec.rb [hunk 2]
```diff
@@ -22,61 +26,57 @@
...
-      visit "/concern/parent/#{parent.id}/generic_works/#{curation_concern.id}/edit"
+      visit "/concern/parent/#{parent.id}/monographs/#{curation_concern.id}/edit"
       click_on "Save"
 
-      expect(parent.reload.ordered_members.to_a.length).to eq 1
-      expect(new_parent.reload.ordered_members.to_a.length).to eq 1
-
-      expect(curation_concern.reload.in_works_ids.length).to eq 2
+      expect(parent.member_ids.count).to eq 1
+      expect(new_parent.member_ids.count).to eq 1
```
Comment (dlpierce) [inline spec/features/create_child_work_spec.rb:66]:
```suggestion
      expect(Hyrax.query_service.find_by(id: parent.id).member_ids.count).to eq 1
      expect(Hyrax.query_service.find_by(id: new_parent.id).member_ids.count).to eq 1
```

File: spec/features/create_child_work_spec.rb [hunk 3]
```diff
@@ -22,61 +26,57 @@
...
-        visit "/concern/parent/#{parent.id}/generic_works/#{curation_concern.id}/edit"
-        first("input#generic_work_in_works_ids", visible: false).set new_parent.id
+        visit "/concern/parent/#{parent.id}/monographs/#{curation_concern.id}/edit"
+        first("input#monograph_in_works_ids", visible: false).set new_parent.id
         first("input#parent_id", visible: false).set new_parent.id
         click_on "Save"
 
-        expect(new_parent.reload.ordered_members.to_a.length).to eq 0
-        expect(page).to have_content "Works can only be related to each other if user has ability to edit both."
+        expect(new_parent.member_ids.count).to eq 0
```
Comment (dlpierce) [inline spec/features/create_child_work_spec.rb:79]:
```suggestion
        expect(Hyrax.query_service.find_by(id: new_parent.id).member_ids.count).to eq 0
```

**Review by dlpierce — CHANGES_REQUESTED** [review]
This will reload the object as in the original spec.

**Comment by bwatson78** [comment]
@dlpierce Change requests addressed in https://github.com/samvera/hyrax/pull/6428/commits/36ecee58dcba6be21d1e152a377650c06f37af7d .

**Review by dlpierce — APPROVED** [review]

**Pull request merged**